code/datums/components/crafting/recipes.dm

```diff
@@ -142,6 +142,28 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_WEAPON
 
+/datum/crafting_recipe/reciever
+	name = "Modular Rifle Reciever"
+	tools = list(TOOL_WRENCH, TOOL_WELDER, TOOL_SAW)
```

Comment:
does any other recipe in the game require TOOL_SAW? as far as I can tell, it's only present in circular saws (from autolathes), chainsaws (which require circular saws to be crafted), and null rods in chainsaw-themed forms (which you'd usually have to mug a chaplain for).

Reply:
I think it is rare yes, but also we lack actual saws and handsaws. Which is kind of unfortunate but yet here we are.